benchmark/utils.go

```diff
@@ -18,16 +18,43 @@ package benchmark
 
 import (
 	"encoding/csv"
+	"encoding/json"
 	"errors"
 	"os"
 	"os/exec"
 )
 
 type ImageDescriptor struct {
-	ShortName            string
-	ImageRef             string
-	SociIndexManifestRef string
-	ReadyLine            string
+	ShortName            string `json:"short_name"`
+	ImageRef             string `json:"image_ref"`
+	SociIndexManifestRef string `json:"soci_index_manifest_ref"`
+	ReadyLine            string `json:"ready_line"`
+}
+
+func GetImageList(file string) ([]ImageDescriptor, error) {
```

Comment:
nit: Might be slightly faster to open the file here and then parse it in their respective functions? Or would attempting to read in csv.NewReader/json.Unmarshal mutate the file descriptor even on failure?

Reply:
Done.